Info-Orbs/include/model/webDataElementModel.h

```diff
@@ -27,7 +27,7 @@ class WebDataElementModel {
     bool isChanged();
     void setChangedStatus(bool changed);
 
-    void parseData(const JsonObject& doc, int32_t defaultColor, int32_t defaultBackground);
+    void parseData(JsonObject doc, int32_t defaultColor, int32_t defaultBackground);
```

Comment:
Here you are possibly creating a copy of the object again, is this intentional?
Depending on how it is implemented this might be either a cheap or an expensive (computational / memory usage) operation